src/pkg/retention/policy/models.go

```diff
@@ -58,6 +60,24 @@ type Metadata struct {
 	Scope *Scope `json:"scope" valid:"Required"`
 }
 
+// ValidateRetentionPolicy validate the retention policy
+func (m *Metadata) ValidateRetentionPolicy() error {
```

Comment:
ditto

Reply:
Same as above. This is because we need to skip `utils.ValidateCronString()` validation for cron with empty string "". In preheat and tag retention, it uses empty string cron to unschedule an existing job.